source/fundamentals/qe.txt

```diff
@@ -0,0 +1,140 @@
+====================
+{+qe+}
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to install and use **{+qe+}**
+in the MongoDB Node.js driver.
+
+{+qe+} allows you to encrypt specific data fields within a document with
+your MongoDB client application before sending the data to the server.
+Starting in MongoDB 6.0 Enterprise and Atlas, you can perform this
+client-side encryption automatically.
+
+With {+qe+}, your client application encrypts fields client-side. {+qe+}
+is useful for situations in which applications must guarantee that
+unauthorized parties, including server administrators, cannot read the
+encrypted data.
+
+This guide is a quick introduction to {+qe+} using the Node.js driver.
+For in-depth information on how {+qe+} works, see the
+:ref:`{+qe+} reference <qe-features>` documentation.
+
+Installation
+------------
+
+To get started with {+qe+} in your client application, you need:
+
+- the MongoDB Node.js driver
+- `mongodb-client-encryption <https://www.npmjs.com/package/mongodb-client-encryption>`__ version ``2.2.0-alpha.1`` or later
```

Comment:
It might be better to just say `2.2.0` here in the docs, we don’t really want people using alpha versions I imagine.